projects/igniteui-angular/src/lib/slider/slider.component.ts

```diff
@@ -700,6 +719,54 @@ export class IgxSliderComponent implements ControlValueAccessor, OnInit, AfterVi
         );
     }
 
+    private resetValue() {
```

Comment:
Reset means setting the value to initial/default value. This method does value validation. So I think a proper name would be `invalidateValue`.